<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addbook</class>
 <widget class="QMainWindow" name="addbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="11" column="0">
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="12" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(226, 226, 226);
	color: rgb(0, 0, 0);
	font: 700 11pt &quot;Segoe UI&quot;;
}</string>
        </property>
        <property name="text">
         <string>Add Book</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>ISBN</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Science Fiction</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fantasy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Novel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Historical</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bioraphy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mystery</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Business</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Childeren</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Comics</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Travel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sprituality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Selfhelp</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Science</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Romance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Religion</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Psychology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Poetry</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Philosophy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Comedy</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(226, 226, 226);
	color: rgb(0, 0, 0);
	font: 700 11pt &quot;Segoe UI&quot;;
}</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Publisher</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Genre</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Discription</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
